### PR TITLE
Propose file change

### PR DESCRIPTION
**Pull request by parthjawale**
Incorrect release version spelling. (An extra 'release'was added after release-1.17 which I changed to 'version' for better consistency)

### CODE REVIEW

**Comment by parthjawale** [comment]
I've signed the CLA. Please verify.

**Comment by parthjawale** [comment]
I've signed the CLA. Please verify.

/check-cla

**Comment by sftim** [comment]
I'm going to remove the emphasis from the PR title, for consistency with other pull requests
/retitle Propose file change

File: content/en/docs/contribute/start.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ to base your work on. Use these guidelines to make the decision:
     to beta in the next minor version, you need to know what the next minor
     version number is.
   - Find the release branch named for that version. For example, features that
-    changed in the v{{< release-branch >}} release got documented in the branch
+    changed in the {{< release-branch >}} version got documented in the branch
     named `dev-{{< release-branch >}}`.
```
Comment (sftim) [inline content/en/docs/contribute/start.md]: I don't think there are any shortcodes that work exactly how things are needed to here, so the easy fix looks like:
```suggestion
    changed in Kubernetes version 1.16 got documented in the branch named
    `dev-1.16`, which was then merged into `master` as part of the release process.
```

Comment (sftim) [reply content/en/docs/contribute/start.md]: Does that make sense @parthjawale ?

Comment (parthjawale) [reply content/en/docs/contribute/start.md]: Yes, sounds perfect.
I'm gonna commit this to my PR.

**Comment by RA489** [comment]
@parthjawale Please resolve the merge conflict to get thsi merge.

**Comment by parthjawale** [comment]
Alright, on it.

**Comment by jimangel** [comment]
@parthjawale let me know if you have any issues resolving conflicts. Thanks!

**Comment by zacharysarah** [comment]
@parthjawale 👋 Please sign the contributor agreement in order to continue.

/close

**Pull request closed without merging**